acra/getting-started/trying/_index.md

```diff
@@ -37,17 +37,17 @@ Now you can connect to (you can see the default DB name and credentials inside t
 | 9191/tcp | AcraServer/AcraConnector API | zonemode                                |
 | 9393/tcp | AcraServer                   | absent AcraConnector                    |
 | 9494/tcp | AcraConnector                | present AcraConnector                   |
-| 8000/tcp | AcraWebconfig                | present AcraConnector and SecureSession |
 | 5432/tcp | PostgreSQL                   | PostgreSQL                              |
 | 3306/tcp | MySQL                        | MySQL                                   |
+| 8000/tcp | AcraWebconfig (deprecated and removed after 0.90.0) | present AcraConnector and SecureSession |
 
 Please refer to the [Launching Acra from Docker images](/acra/getting-started/installing/launching-acra-from-docker-images/) ff you need more information about Docker images.
 
 ### Compose files
 
 We want you to be able to easily try the most useful schemes that we prepared as Docker Compose files in the `docker` subdirectory. The name of each Docker Compose file describes its components and their interconnections in a simple form. For example: `docker-compose.pgsql-nossl-server-ssession-proxy.yml` is a scheme with Postgresql DB, AcraServer, and AcraConnector, connected to AcraServer through the Themis Secure Session link.
 
-The examples contain references to `acra-keymaker` and `acra-authmanager` containers inside. They are used for creation and distribution of the necessary keys. They were included for simplification of the test launch and should not be used in production schemes (where the keys should be generated manually and deployed to an appropriate host according to the security rules of your infrastructure).
+The examples contain references to `acra-keymaker` and `acra-authmanager` (deprecated and removed after 0.90.0) containers inside. They are used for creation and distribution of the necessary keys. They were included for simplification of the test launch and should not be used in production schemes (where the keys should be generated manually and deployed to an appropriate host according to the security rules of your infrastructure).
```

Comment:
```suggestion
The examples contain references to `acra-keymaker` and `acra-authmanager` (deprecated and removed since 0.91.0) containers inside. They are used for creation and distribution of the necessary keys. They were included for simplification of the test launch and should not be used in production schemes (where the keys should be generated manually and deployed to an appropriate host according to the security rules of your infrastructure).
```